### DISCUSSION TITLE
Make extension version visible in new extension panel

### DISCUSSION THREAD

**Discussion by that1matt** [directus/directus#20281] [Feature Requests]
### Summary

Currently when making a change to an extension, and pushing to Directus Self-Hosted/Directus Enterprise, we have no way of knowing if our change has made it to production.

With Directus 10.7.1, and the new extensions panel in settings, it should show the version that is installed.

### Basic Example

![Screenshot 2023-10-31 at 1 48 42 PM](https://github.com/directus/directus/assets/9679004/6ff87a2f-cfe2-479a-9eac-3f02a5442340)

### Motivation

When making changes to extensions, and when deploying the new changes there is no easy way to validate if the new version has been installed. This is especially true for Directus Enterprise.

### Detailed Design

Would need to add a new field to the `directus_extensions` collection for `version`. 
Then along with getting the `name` from the `package.json`, getting the current `version`, and updating the `version` field in `directus_extensions`

### Must Have:
- Visible output of currently "successfully" installed extension visible in Extensions panel in settings.

### Should Have:
- In the more menu (triple dots), could have a link to render any issues with installation.

### Drawbacks

Would possibly need to find a way to surface any installation issues if latest version couldn't be installed.

### Alternatives

Currently, you could have a custom endpoint with an unauthenticated endpoint return the version.

### Adoption Strategy

A simple migration would solve this issue, but extending the current `directus_extensions` collection to allow for a version field.

**Comment by rijkvanzanten** [comment]
Yes 100%!

**Reply by rijkvanzanten** [reply]
This is a no-brainer to me. Tracking the implementation here: https://github.com/directus/directus/issues/20282

**Comment by rijkvanzanten** [comment]
This was implemented and released in 10.8 🙂
